bigtable-dataflow-parent/bigtable-beam-import/src/main/java/com/google/cloud/bigtable/beam/sequencefiles/ImportJob.java

```diff
@@ -109,10 +109,11 @@
     @SuppressWarnings("unused")
     void setSourcePattern(ValueProvider<String> sourcePath);
 
-    @Description("Optional BigTable write throttling in milliseconds")
-    ValueProvider<String> getBigtableWriteThrottleMs();
+    @Description("Optional Enable BigTable write throttling. Value in milliseconds.")
```

Comment:
The way we are using this flag and naming can be *very* confusing.
I would add more description here to help clarify what we are trying to do.

This field really is "mutation throttling latency threshold in milliseconds".
When the threshold is set to a value that is greater than 0, mutation throttling will be enabled and mutations will be throttled on the client side if the write latency has reached the threshold.